From 0d2515090adbf1ed589046047727ce9a088c0db2 Mon Sep 17 00:00:00 2001
From: Adrien Destugues <pulkomandy@pulkomandy.tk>
Date: Sun, 12 Jan 2014 18:36:42 +0100
Subject: Hack to avoid hardcoded library search path.

Makes it possible to find zlib when building as an hybrid package.

diff --git a/config.lib b/config.lib
index 2dc14a8..215705d 100644
--- a/config.lib
+++ b/config.lib
@@ -2597,10 +2597,10 @@ detect_library() {
 			fi
 		fi
 		if [ -z "$res" ] && [ "$os" = "HAIKU" ]; then
-			eval "$2=`ls -1 /boot/common/include/$4*.h 2>/dev/null | egrep \"\/$5\$\"`"
+			eval "$2=`ls -1 $includeDir/$4*.h 2>/dev/null | egrep \"\/$5\$\"`"
 			eval "res=\$$2"
 			if [ -z "$res" ]; then
-				log 2 "  trying /boot/common/include/$4$5... no"
+				log 2 "  trying $includeDir/$4$5... no"
 			fi
 		fi
 
-- 
1.8.3.4

